kovri.md

```diff
@@ -1,55 +1,51 @@
-Currently based on the Invisible Internet Project's open specifications, Kovri uses both garlic encryption and garlic routing to create a private, protected overlay-network across the internet.
-This overlay-network provides Alice and Bob with the ability to effectively hide their geographical location and internet IP address.
-Essentially, Kovri *covers* an application's internet traffic to make it anonymous within the network.
-Kovri is lightweight, security-focused, and fully compatible with the I2P network.
+## 4) Examples
 
-If Alice sends her transaction with Kovri, nodes that log Monero traffic will no longer see her internet IP address, rendering passive surveillance tactics close to useless.
+Suppose Alice wants to send a transaction to Bob.
+She generates a transaction and in order for her transaction to have the greatest chance of being included in the next block, she needs to reach as many nodes as possible.
+Alice connects to five nodes and these nodes then forward the transaction to other nodes, which in turn rebroadcast it repeatedly until many (or all) nodes in the network have this transaction.
+Unfortunately, Alice takes a risk in broadcasting her transaction.
+Some nodes may be logging IP addresses when transaction requests are received.
+Even though wallet addresses and the amount of Monero remain private, an attacker with enough resources could attempt to associate transactions with IP addresses to determine where transactions originate from.
+This could potentially lead to Alice receiving a knock on the door from Eve or her transaction being blocked from the rest of the network.
+So what can Alice do to mitigate these threats? Well, she can use Kovri!
+If Alice sends her transaction with Kovri, nodes will no longer see her IP address, making passive surveillance tactics impractical.
 
 Now let’s imagine a different scenario.
-Suppose Alice has an internet service provider that does not want her to use Monero, and send her a warning to cease and desist.
-Alice can optionally send all the Monero sync data, not just the transaction broadcast, through Kovri.
-Now, her Monero connections are hidden from her ISP.
-This improves Monero's censorship resistance, since other people no longer know who is running a Monero node, and others cannot demand that these nodes be shut down.
-
-Let’s look at one last scenario.
-Suppose Charlie is the operator of a mining pool that controversially donates a portion of the block reward to a political party.
-Other nodes could purposefully reject his solved blocks to express their disagreement with this policy.
-This sort of attack is called a Sybil attack.
-However, if Charlie can send his solved blocks through Kovri, then no one can censor the work from his pool, since they no longer know that his pool created it.
+Suppose Charlie has an Internet Service Provider that does not want him to use Monero and he receives a warning to cease and desist in the mail.
```

Comment:
>Suppose Charlie is the operator of a mining pool
>Suppose Charlie has an Internet Service Provider

Charlie was specifically an operator of a mining pool. In your revision, Charlie has no context; he could be anyone. This is not the definition of a Charlie as 3rd person in relation to Alice and Bob (as was the operator of a mining pool).

This sentence is also seems more relevant for Alice who runs her own node and uses her wallet locally, yes? And/or Bob who syncs nodes with other Bobs?